main.tex

```diff
@@ -0,0 +1,123 @@
+available. The authors are interested in designing and devising a reference
+implementation available as opensource software with materials available under
+permissive sharing licenses such as creative commons.
+\end{abstract}
+
+\begin{IEEEkeywords}
+Holography, image analysis.
+\end{IEEEkeywords}
+
+% Main section includes
+\input{sections/objectives}
+
+% Include all citations in the database
+\nocite{*}
+
+% Some example resources
+\section{Examples}
+
+\begin{figure}[t]
+\centering
+\includegraphics[width=200px]{figures/julian-harty.eps}
+\caption{(a) Diagram of the experimental set up. (b) Detail of the test object used.}
+\label{fig_env1}
+\end{figure}
+
+\begin{equation}
+x=\sum\limits_{1=0}^z 2^1Q
+\label{eq1}
+\end{equation}%
+
+\begin{table}[!t]
+\centering
+\caption{Math Spacings Used By \LaTeX}
+\label{tab1}
+\begin{IEEEeqnarraybox}[\IEEEeqnarraystrutmode\IEEEeqnarraystrutsizeadd{2pt}{1pt}]{v/c/v/r/v/c/v/c/v/c/v}
+\IEEEeqnarrayrulerow\\
+& \mbox{{\bf Size}} && \mbox{{\bf Width}} && \mbox{{\bf Cmd.}} &&
+\mbox{{\bf Used for}} && \mbox{{\bf Example}} &\\
+\IEEEeqnarraydblrulerow\\
+\IEEEeqnarrayseprow[3pt]\\
+& \mbox{small} && \mbox{1/6 em} && \verb+\,+ && \mbox{symbols} && a b &\IEEEeqnarraystrutsize{0pt}{0pt}\\
+\IEEEeqnarrayseprow[3pt]\\
+\IEEEeqnarrayrulerow\\
+\IEEEeqnarrayseprow[3pt]\\
+& \mbox{medium} && \mbox{2/9 em} && \verb+\:+ && \mbox{binary operators} && a + b &\IEEEeqnarraystrutsize{0pt}{0pt}\\
+\IEEEeqnarrayseprow[3pt]\\
+\IEEEeqnarrayrulerow\\
+\IEEEeqnarrayseprow[3pt]\\
+& \mbox{large} && \mbox{5/18 em} && \verb+\;+ && \mbox{relational operators}&& a = b &\IEEEeqnarraystrutsize{0pt}{0pt}\\
+\IEEEeqnarrayseprow[3pt]\\
+\IEEEeqnarrayrulerow\\
+\IEEEeqnarrayseprow[3pt]\\
+& \mbox{negative small} && \mbox{${-}$1/6 em} && \verb+\!+ && \mbox{misc. uses} && ab &\IEEEeqnarraystrutsize{0pt}{0pt}\\
+\IEEEeqnarrayseprow[3pt]\\
+\IEEEeqnarrayrulerow
+\end{IEEEeqnarraybox}
+\end{table}
+
+\begin{itemize}
+\item Weight parameters for the simulated robot.
+\item Length of each body link.
+\item Specifications of body joints. Upper Torso means the spine. Due to symmetry, body parts from the right are not shown.
+\item Neuron parameters. $\Theta$ is the threshold, $\Gamma$ is the gain. $\tau_{D}$ and $\tau_{A}$ are respectively the time constant of the dendritic sums and that of the frequency adaptation. $\mu$ is the coefficient of frequency adaptation.
+\end{itemize}
+Holograms recorded in such a fashion can not be reconstructed in the conventional way with an optical reconstruction beam. In order to numerically reconstruct the holograms, the surface modulation was digitized with a Novascan atomic force microscope (AFM) operated in tapping mode. Holograms recorded in such a fashion can not be reconstructed in the conventional way with an optical reconstruction beam. In order to numerically reconstruct the holograms, the surface modulation was digitized with a Novascan atomic force microscope (AFM) operated in tapping mode. 
```

Comment:
please remove other folk's text, random text would be better from a copyright perspective...